composer.json

```diff
@@ -5,7 +5,7 @@
 	"homepage": "https://www.netresearch.de",
 	"license": ["GPL-2.0-or-later"],
 	"require": {
-		"typo3/cms-core": "^10.4"
+		"typo3/cms-core": "^10.2"
```

Comment:
Version changes seems unrelated to "[BUGFIX] #71: Fix tsconfig includes #99" for me.

Reply:
Right, removed.